trp_use_case/view/qweb_report_use_case_collection.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <template id="qweb_report_ucc">
+                <t t-foreach="docs" t-as="o">
+                    <t t-call="report.layout">
+                        <div class="page">
+                            <div class="row">
+                                <div class="col-xs-12">
+                                    <h2><b><span t-field="o.name" />,</b></h2>
+                                </div>
+                            </div>
+                            <div class="row">
+                                <div class="col-xs-5">
+                                    For <span t-field="o.partner_id.name" />
+                                </div>
+                            </div>
+                            <br/>
+                            <div class="row">
+                                <div class="col-xs-4">
+                                    <h4><b><i>Revisions</i></b></h4>
+                                </div>
+                            </div>
+                            <t t-foreach="o.version_tag_ids" t-as="u">
+                                <div class="col-xs-12">
+                                    <span t-field="u.date"/>
+                                    <span t-field="u.user" /> :
+                                    <span t-field="u.name" />
+                                </div>
+                            </t>
+                            <br/>
+                            <div class="row">
+                                <div class="col-xs-4">
+                                    <h4><b><i>Introduction</i></b></h4>
+                                </div>
+                            </div>
+                            <div class="row">
+                                <div class="col-xs-12">
+                                    <span t-field="o.description" />
+                                </div>
+                            </div>
+                            <br/>
+                            <div class="row">
+                                <div class="col-xs-4">
+                                    <h4><b><i>Use-cases</i></b></h4>
+                                </div>
+                            </div>
+                            <t t-foreach="o.use_case_ids" t-as="ui">
+                                <table class="table table-bordered table-striped" style="page-break-inside:avoid">
+                                    <col width="25%" />
+                                    <col width="75%" />
```

Comment:
those would belong within a `colgroup` (https://www.w3.org/TR/html5/tabular-data.html#the-colgroup-element) element, but all of this is deprecated anyways. Use bootstrap's `col-md-*` classes: http://getbootstrap.com/css/#grid